LamestWebserver/UnitTests/LoggerTests.cs

```diff
@@ -0,0 +1,78 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using LamestWebserver.Core;
+using System.IO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTests
+{
+    [TestClass]
+    public class LoggerTests
+    {
+        private const string PATH_TO_NEW_FILE = "newFile.log";
+        private const string Pat_TO_PARALLEL_FILE = "parallelLog.log";
```

Comment:
C# coding convention for constants is:
``` c#
public const Min = 0;
```
[see this. 😉 ](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)

Reply:
i blame java for this

Reply:
😄